lib/next_rails/bundle_report.rb

```diff
@@ -58,6 +59,45 @@ def self.gem_header(_gem)
       header
     end
 
+    def self.compatible_ruby_version(rails_version)
+      # find all the versions of rails gem
+      uri = URI('https://rubygems.org/api/v1/versions/rails.json')
+      res = Net::HTTP.get_response(uri)
+      all_versions_res = JSON.parse(res.body)
+
+      # push all the versions in an array
+      all_versions = []
+      all_versions_res.each { |rv| all_versions << rv['number'] }
+
+      rv = rails_version[:rails_version]
+      matched_versions = all_versions.select { |h| h.match(rv) && h.start_with?(rv) }
```

Comment:
I think `h.start_with?(rv)` is a subset of `h.match(rv)` (I mean, if it starts with rv it will always match rv, so we can just check the `start_with?` here I think)